src/backend/controllers/userController.js

```diff
@@ -277,9 +276,9 @@ exports.confirmToken = async (req: Request, res: Response) => {
 			return res
 				.status(400)
 				.json({ type: 'not-verified',
-					msg: 'The token you are using is not a valid token. ' +
+					msg: 'The token you are using is not valid. ' +
 					'Check your confirmation email and try again. ' + 
-					'Otherwise, your token may have expired.' });
+					'Otherwise, request a new confirmation email at the Login page.' });
```

Comment:
I think it would be better to remove the "Otherwise," here and just say "Request a new....."